0.preprocess-sites/README.md

```diff
@@ -0,0 +1,33 @@
+# Preprocessing Sites
+
+In this first module of the recipe, we perform a series of steps to prepare data for profile generation.
+The `preprocessing_config.yaml` controls all the pertinent details for this module.
+
+<p align="center">
+<img src="https://raw.githubusercontent.com/broadinstitute/pooled-cell-painting-profiling-recipe/82eaf532e7a3ab145c4b821268c13c531b693dcb/0.preprocess-sites/media/preprocessing_workflow.png" width="500">
```

Comment:
updated! Thanks for this suggestion

![preprocessing_workflow](https://user-images.githubusercontent.com/7340421/87830357-dbde3a80-c84e-11ea-8340-0d920b8184ea.png)

Look ok?

Reply:
maybe `Image metadata` instead of `Image Data`

Reply:
You mean the input into 4.image-qc? I think `Image data` is appropriate because it does include actual measurements being made from each image/site, even if they aren't the measurements used for generating profiles. Though I guess it depends on where you want to draw the line between data/metadata? If you use `data` to mean "the data used to generate profiles" and everything else is therefore `metadata`, then the switch would be appropriate.

Reply:
as long as it's not blatantly wrong, I am ok keeping as is. I will merge!